apps/examples/webserver_test/webserver_test_main.c

```diff
@@ -0,0 +1,1308 @@
+
+
+#define MBED_DEBUG_LEVEL 1
+#define WS_TEST_BUF_SIZE 4600
+
+#define WGET_OK                    0
+#define WGET_ERR                   -1
+#define WGET_MSG_CONSTRUCT_ERR     -2
+#define WGET_SOCKET_CONNECT_ERR    -3
+
+#define HTTPSTATUS_NONE            0
+#define HTTPSTATUS_OK              1
+#define HTTPSTATUS_MOVED           2
+#define HTTPSTATUS_ERROR           3
+
+#define ISO_nl                     0x0a
+#define ISO_cr                     0x0d
+#define ISO_space                  0x20
+
+#define WS_TEST_MAXHTTPLINE 200
+#define WS_TEST_MAXMIMESIZE 32
+#define WS_TEST_MAXHOSTNAME 128
+#define WS_TEST_MAXFILENAME 128
+
+#define PRNT(...) \
+	do { \
+		printf("[WS_TEST]%s():%d: ", __func__, __LINE__); \
+		printf(__VA_ARGS__); \
+		printf("\n"); \
+	} while(0)
+
+#define FUNC_EN PRNT("entry")
+#define FUNC_EX PRNT("exit")
+
+
+typedef struct ws_test_wget_s {
+	/* Internal status */
+	uint8_t state;
+	uint8_t httpstatus;
+	uint16_t port; /* The port number to use in the connection */
+	/* These describe the just-received buffer of data */
+	FAR
+	char *buffer; /* user-provided buffer */
+	int buflen; /* Length of the user provided buffer */
+	int offset; /* Offset to the beginning of interesting data */
+	int datend; /* Offset+1 to the last valid byte of data in the buffer */
+	/* Buffer HTTP header data and parse line at a time */
+	char line[WS_TEST_MAXHTTPLINE];
+	int ndx;
+#ifdef CONFIG_WEBCLIENT_GETMIMETYPE
+	char mimetype[WS_TEST_MAXMIMESIZE];
+#endif
+	char hostname[WS_TEST_MAXHOSTNAME];
+	char filename[WS_TEST_MAXFILENAME];
+} ws_test_wget_s;
+
+
+static const char g_http10[] = "HTTP/1.0";
+static const char g_http11[] = "HTTP/1.1";
+#ifdef CONFIG_WEBCLIENT_GETMIMETYPE
+static const char g_httpcontenttype[] = "content-type: ";
+#endif
+static const char g_httphost[] = "host: ";
+static const char g_httpget[] = "GET ";
+static const char g_httppost[] = "POST ";
+static const char g_httpput[] = "PUT ";
+static const char g_httpdelete[] = "DELETE ";
+static const char g_httpcrnl[] = "\r\n";
+
+static const char g_httpform[] = "Content-Type: application/x-www-form-urlencoded";
+static const char g_httpcontsize[] = "Content-Length: ";
+static const char g_httpchunked[] = "Transfer-Encoding: chunked";
+static const char g_trailerheader[] = "Trailer: Expires";
+static const char g_expires[] = "Expires: Fri, 21 Oct 2022 07:28:00 GMT";
+static const char *g_tlsname = "ws_test_tls_client";
+
+static int g_keep_alive_count = 0;
+static int g_chunk_count = 0;
+static int g_trailer = 0;
+static int g_normal_clen = 0;
+static int g_last_clen = 0;
+static int g_https = 0;
+static int g_tc_pass = 0;
+static int g_is_test_entity = 0;
+static const char HEADERFIELD_CONNECT[] = "Connection";
+static const char HEADERFIELD_CLOSE[] = "close";
+static const char HEADERFIELD_KEEPALIVE[] = "keep-alive";
+static const char HEADERFIELD_USERAGENT[] = "User-Agent";
+static const char HEADERFIELD_TINYARA[] = "TinyARA";
+
+
+static const char c_ca_crt_ec[] =
```

Comment:
Please prepare PR to use same set of certificates in examples/webserver test application.

Reply:
done